README.md

```diff
@@ -42,6 +42,8 @@ and
 [acknowledgements](https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/Acknowledgements.html)
 page for more details.  
 
+If you want to [CONTRIBUTE](./HOW_TO_CONTRIBUTE.md )!
```

Comment:
keep in mind that, currently, people external to the organization cannot create a PR and merge it because PRs created from forks cannot run the CI tests and will never be merged.